core/src/main/java/org/apache/spark/memory/MemoryConsumer.java

```diff
@@ -54,7 +54,7 @@ public MemoryMode getMode() {
   /**
    * Returns the size of used memory in bytes.
    */
-  protected long getUsed() {
+  public long getUsed() {
```

Comment:
This is exposed so it can be used in `AbstractBytesToBytesMapSuite` to free the actual allocated memory (which is hard to calculate at the `free()` call-site because `spill()` may have been called).